portfolio/src/main/webapp/style.css

```diff
@@ -99,4 +142,26 @@ margin-top: -10px;
 /* images in "Pictures of My Life" section */
 img.mylife-pics{
     width: 100%;
-}
\ No newline at end of file
+}
+
+
+/*  External Citation:
+    https://stackoverflow.com/questions/52321212/iphone-xr-xs-xs-max-css-media-queries
+ */
+/* 1792x828px at 326ppi */
+@media only screen 
```

Comment:
I don't have time right now to search in depth and exactly advise on best way to make the screen responsive to diff sizes, but you can try use less restrictive conditionals here. Looks like you're specifying the exact width and height of a screen for your rule to trigger.

Maybe try something like this:

https://www.w3schools.com/cssref/css3_pr_mediaquery.asp

i.e. if screen width is less than X, trigger this rule

Reply:
thanks for the suggestion!